lib/src/modules/compiler/template_parser/GRAMMAR.md

```diff
@@ -0,0 +1,38 @@
+#Template Grammar
+
+###Todos
+* Can you interpolate comments?
```

Comment:
Good point. I imagine it could be useful for debugging in a QA environment. I don't think it is supported yet.